src/object.c

```diff
@@ -776,7 +783,6 @@ size_t objectComputeSize(robj *o, size_t sample_size) {
                 ele = dictGetKey(de);
                 ele2 = dictGetVal(de);
                 elesize += sdsAllocSize(ele) + sdsAllocSize(ele2);
-                elesize += sizeof(struct dictEntry);
```

Comment:
After checking, I can't agree with you more.
But in the mean time, I found that while computing the size of dict,
something is missing, which is the size of dictht.
Fixed that.